app/views/govuk_publishing_components/components/related_navigation/_section.html.erb

```diff
@@ -60,7 +60,7 @@
         </a>
       </li>
 
-      <li class="gem-c-related-navigation__link">
+      <li class="gem-c-related-navigation__link gem-c-related-navigation__link--toggle-wrap">
```

Comment:
There's a `toggle-wrap` class still on line 51 and I think it's unused - can you check if we can take it out?

Reply:
I don't think we can because the `toggle-wrap` class is JavaScript hook. The extra class I added is for styling - it's badly named, so I'll rename it to something clearer.

Reply:
Oh, `toggle-wrap` is a JS hook? Would you mind prefixing it with `js-`? Might save me repeating this confusion later 😄